src/dmd/compiler.d

```diff
@@ -48,7 +48,7 @@ extern (C++) __gshared
  * A data structure that describes a back-end compiler and implements
  * compiler-specific actions.
  */
-struct Compiler
+extern (C++) struct Compiler
```

Comment:
Please split these `extern(C++)` attribute off into a separate PR, also make sure that they are up to date and include e.g. [TargetCPP](https://github.com/dlang/dmd/blob/master/src/dmd/target.d#L738) and [TargetC](https://github.com/dlang/dmd/blob/master/src/dmd/target.d#L659)

Reply:
Can you please elaborate on this? What do you mean by up to date?
Thank you

Reply:
I mean the above two linked structs were added, your diff does not account for them and there may be more that were added.

Reply:
@thewilsonator please see [PR](https://github.com/dlang/dmd/pull/10321)

Reply:
Thanks.